pkg/metrics/metrics.go

```diff
@@ -18,13 +20,14 @@ import (
 )
 
 var (
-	registry            *prometheus.Registry
-	registryOnce        sync.Once
-	metricsWithPod      []*prometheus.MetricVec
-	metricsWithPodMutex sync.RWMutex
-	podQueue            workqueue.DelayingInterface
-	podQueueOnce        sync.Once
-	deleteDelay         = 1 * time.Minute
+	registry            	*prometheus.Registry
+	registryOnce        	sync.Once
+	metricsWithPod      	[]*prometheus.MetricVec
+	metricsWithPodMutex 	sync.RWMutex
+	podQueue            	workqueue.DelayingInterface
+	podQueueOnce        	sync.Once
+	deleteDelay         	= 1 * time.Minute
```

Comment:
I'll restore this spacing -- at one point I had introduced a variable here but later removed it.

Reply:
Ran go fmt.